packages/graphql/analysis_options.yaml

```diff
@@ -0,0 +1,6 @@
+include: package:pedantic/analysis_options.yaml
+
+analyzer:
+  exclude:
+    - test/_data/**
+    - lib/**/**
```

Comment:
is this to temporarily mute some error output? Looks pretty awkward 😅

Reply:
Yea, it is temporary. As long as document is marked for deprecation then the linting test will not pass.

Reply:
Can’t we override a specific listing rule?